### PR TITLE
Clicking on a dot reveals the attatched row

### PR DESCRIPTION
**Pull request by AdaRoseCannon**

### CODE REVIEW

**Comment by AdaRoseCannon** [comment]
Fixes #6

**Comment by AdaRoseCannon** [comment]
Tempted also add the exposed table columns as additional labels which get revealed on hover.

or could just set the title attribute.

File: client/js/lib/d3.js [hunk 1]
```diff
@@ -92,13 +92,22 @@ module.exports = function ({
...
+		const row = document.getElementById(d.name);
+		if (!row) return;
+		row.classList.toggle('collapsed');
+	}
+
 	node.append('circle')
 		.attr('class', 'node')
 		.attr('r', 8)
-		.attr('id', n => `${n.name}--graph-point`)
+		.attr('id', n => `${n.name}--graph-point`)
```
Comment (seanmtracey) [inline client/js/lib/d3.js]: Whitespace

**Comment by seanmtracey** [comment]
LGTM and tested locally. Merged :)

**Pull request merged**